lib/internal/test_runner/mock/mock_timers.js

```diff
@@ -8,36 +8,37 @@ const {
   ArrayPrototypeAt,
   ArrayPrototypeForEach,
   ArrayPrototypeIncludes,
-  DateNow,
   FunctionPrototypeApply,
   FunctionPrototypeBind,
   Promise,
   SymbolAsyncIterator,
   SymbolDispose,
   globalThis,
+  MathAbs,
```

Comment:
they should be in alphabetic order (MathAbs comes after FunctionPrototypeBind). run `make lint` (it should tell about this rule)

Reply:
I ran the `make lint` and  `make lint-js-fix` but it didn't change. I'll do it by hand!